tensor2tensor/data_generators/translate_mnzh.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Created by Jack on 04/02/2018
+
+# coding=utf-8
+# Copyright 2017 The Tensor2Tensor Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Data generators for translation data-sets."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+# Dependency imports
+
+from tensor2tensor.data_generators import generator_utils
+from tensor2tensor.data_generators import problem
+from tensor2tensor.data_generators import text_encoder
+from tensor2tensor.data_generators import translate
+from tensor2tensor.utils import registry
+
+import tensorflow as tf
+
+FLAGS = tf.flags.FLAGS
+
+# End-of-sentence marker.
+EOS = text_encoder.EOS_ID
+
+
+@registry.register_problem
+class TranslateEnzhWmt32k(translate.TranslateProblem):
+  """
+    Problem spec for Mn_unicode-zh translation.
+  """
+  @property
+  def targeted_vocab_size(self):
+    return 2**15  # 32k
+
+  @property
+  def source_vocab_name(self):
+    return "vocab.%d.ch.txt" % self.targeted_vocab_size
```

Comment:
why "ch"?  ISO_639-1 code for Chinese is "zh".

Reply:
Sorry, you are right. I just picked the top two characters of " Chinese " then, and did not think too much about this. Sorry, I will make it right.Thank you.